storage/store.go

```diff
@@ -253,8 +260,48 @@ func (si *storeRangeIterator) Reset() {
 	si.index = 0
 }
 
-// StoreConfig contains various parameters of a Store.
-type StoreConfig struct {
+// A Store maintains a map of ranges by start key. A Store corresponds
+// to one physical device.
+type Store struct {
+	*StoreFinder
+
+	Ident          proto.StoreIdent
+	ctx            StoreContext
```

Comment:
Only reason I didn't do that is because that makes all its contents exported on Store. If that's ok I'll go for it.